server/src/routes/posts.js

```diff
@@ -52,4 +53,22 @@ router.delete('/:id', async (req, res) => {
   }
 });
 
+// Commenting once and updating post schema with comment id
+// eslint-disable-next-line no-unused-vars
+router.post('/:id/comment', async (req, res) => {
+  const comment = new Comment({
+    comment_id: req.body.comment_id,
+    body: req.body.body,
+  });
+  try {
+    const newComment = await comment.save();
+    res.status(201).json(newComment);
```

Comment:
Returning newComment in the response isn't a part of the API, if it's possible could this be corrected to reflect the API documentation.

Reply:
Also I believe the response should be sent as the last line of the 'try' statement.

Reply:
should this also go after the update post?

Reply:
How would commnets be stored to the db then?

Reply:
Line 64, where you call comment.save() is when it actually saves into the DB.